### PR TITLE
feat(builder): add git push lock timeout env to builder rc

### PR DESCRIPTION
**Pull request by smothiki**
timeout int in minutes

### CODE REVIEW

File: workflow-dev/tpl/deis-builder-rc.yaml [hunk 1]
```diff
@@ -32,6 +32,8 @@ spec:
               value: "2223"
             - name: BUILDER_STORAGE
               value: "{{ or (env "STORAGE_TYPE") (.storage)}}"
+            - name: "GIT_LOCK_TIMEOUT"
+              value: "10"
```
Comment (sgoings) [inline workflow-dev/tpl/deis-builder-rc.yaml:37]: Seconds? Minutes?

Comment (smothiki) [inline workflow-dev/tpl/deis-builder-rc.yaml:37]: minutes

Comment (sgoings) [inline workflow-dev/tpl/deis-builder-rc.yaml:37]: Maybe add a comment in this area with that info?

Comment (smothiki) [inline workflow-dev/tpl/deis-builder-rc.yaml:37]: A lock makes sure that we are not pushing concurrently to the same GIT repo untill the previous one is finished which takes of the order minutes

**Pull request merged**